docs/Model_Assumptions.rst

```diff
@@ -2,6 +2,27 @@
+
+* **Project lifetime**: The simulation has a defined project lifetime, for which continuous operation is assumed - which means that the first year of operation is exactly like the last year of operation. Existing and optimized assets have to be replaced to make this possible.
+* **Simulation duration**: It is advisable to simulate whole year to find the most suitable combination of energy assets for your system. Sometimes however you might want to look at specific seasons to see their effect - this is possible in the MVS by choosing a specific start date and simulation duration.
+* **Asset costs**: Each asset can have development costs, specific investment costs, specific operation and management costs as well as dispatch costs.
+    * *Replacement costs* are calculated based on the lifetime of the assets, and residual values are paid at the end of the project.
+    * *Development costs* are costs that will occurr regardless of the installed capacity of an asset - even if it is not installed at all. It stands for system planning and licensing costs. If you have optimized your energy system and see that an asset might not be favourable (zero optimized capacities), you might want to run the simulation again and remove the asset, or remove the development costs of the asset.
+    * *Specific investment costs* and *specific operation and maintenance costs* are used to calculate the annual expenditures that an asset has per year, in the process also adding the replacement costs.
+    * *Dispatch price* can often be set to zero, but are supposed to cover instances where utilization of an asset requires increased operation and maintenance or leads to wear.
+* **Pre-existing capacities**: It is possible to add assets that already exist in your energy system with their capacity and age.
+    * *Replacements* - To ensure that the energy system operates continously, the existing assets are replaced with the same capacities when they reached their end of life within the project lifetime.
+    * *Replacement costs* are calculated based on the lifetime of the asset in general and the age of the pre-existing capacities
+* **Fix project costs**: It is possible to define fix costs of the project - this is important if you want to compare different project locations with each other. You can define...
+    * *Development costs*, which could for example stand for the cost of licenses of the whole energy system
+    * *(Specific) investment costs*, which could be an investment into land or buildings at the project site. When you define a lifetime for the investment, the MVS will also consider replacements and reimbursements.
+    * *(Specific) operation and management costs*, which can cover eg. the salaries of at the project site
+
+
```

Comment:
I know this isn't part of this PR, but I miss an explanation about economic calculations, e.g. how is total annuity calculated? But might as well be that I've just overlooked that part.

Reply:
It is in https://multi-vector-simulator.readthedocs.io/en/latest/Model_Equations.html#economic-dispatch.